modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,48 @@
+
+| Apache Kafka | link:fluent-plugin-kafka[fluent-plugin-kafka] | link:https://github.com/fluent/fluent-plugin-kafka/releases/tag/v0.13.1[0.13.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/kafka/broker.go#L141[kafka 2.4.1]
+
+|====
+
+[role="_Where"]
+.Where
+
+Output type:: The target log collector type or protocol.
+Plugin:: The name of the tested plugin or component, which also links to the plugin website.
+Plugin version:: The version of the tested plugin or component, which also links to the version information on the plugin website or link:https://rubygems.org/gems/[rubygems.org]
+Tested with:: The target log collector, protocol, or standard, and their versions, with which Red Hat tests the plugin. These items link to link:https://github.com/openshift/origin-aggregated-logging/blob/release-5.0/fluentd/Gemfile.lock[the corresponding line] of the test configuration file that Red Hat OpenShift Logging version 5.0 uses. This file is the authoritative source of information about which items and versions Red Hat tests and supports.
+
+[NOTE]
+====
+The plugins in the preceding table support a more comprehensive range of target logging collector types and versions than the ones listed in the "Tested with" column. However, Red Hat has higher confidence in the items listed in the "Tested with" column.
+====
+
+[role="_Additional_Comments"]
+.Additional Comments
+
+* Previously, cluster logging used the link:https://rubygems.org/gems/fluent-plugin-remote-syslog[fluent-plugin-remote-syslog] plugin. Now, Red Hat uses the link:https://rubygems.org/gems/fluent-plugin-remote_syslog[fluent-plugin-remote_syslog] plugin, which also supports RFC-5424.
+* Splunk has various ways to ingest syslog data. For more information, see link:https://www.splunk.com/en_us/blog/tips-and-tricks/splunk-connect-for-syslog-turnkey-and-scalable-syslog-gdi.html[Splunk Connect for Syslog: Turnkey and Scalable Syslog GDI - Part 1] and link:https://docs.splunk.com/Documentation/Splunk/latest/Data/Monitornetworkports[Get data from TCP and UDP ports].
+* Logstash can ingest log data from fluent forward v1. For more information, see link:https://www.elastic.co/guide/en/logstash/current/plugins-codecs-fluent.html[Fluent codec plugin].
+** Red Hat OpenShift Logging version 5.0 does not include tests for Logstash.
+** For Logstash to ingest data from Red Hat OpenShift Logging version 5.0, you must configure Logstash with `{ nanosecond_precision => true }` as described in link:https://issues.redhat.com/browse/LOG-962?focusedCommentId=15414482&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15414482[LOG-962 - Add test to verify LF to Logstash over forward protocol].
+* According to this link:https://github.com/zendesk/ruby-kafka#compatibility[compatibility list for ruby-kafka], the fluent-plugin-kafka plugin supports Kafka version 0.11.
```

Comment:
We avoid linking to GitHub repos and links require special permission from Support. I would drop the link and just include the supportability statement by itself.